eternalcore-core/src/main/java/com/eternalcode/core/feature/warp/WarpConfigEntry.java

```diff
@@ -0,0 +1,20 @@
+package com.eternalcode.core.feature.warp;
+
+import com.eternalcode.commons.bukkit.position.Position;
+import net.dzikoysk.cdn.entity.Contextual;
+
+import java.util.List;
+
+@Contextual
+public class WarpConfigEntry {
```

Comment:
What is the difference between `WarpConfigEntry` and `WarpImpl` ? Why are we using both when this could be one class?